### ISSUE TITLE
Download archive not listed in assets under v53.0.3 release

### ISSUE THREAD

**Issue by pythoninthegrass** [forcedotcom/dataloader#402]
Hi there 👋 

**tl;dr** Please return to adding archives to the assets section 🙏 

Myself and likely dozens — if not hundreds — of Mac admins rely on Github API listings to check for new releases to applications such as dataloader and subsequently download based on the key-value pair beginning with `browser_download_url`. We use an open-source shell script called [Installomator](https://github.com/Installomator/Installomator) to largely automate this process.

In the latest release, `curl --silent --fail https://api.github.com/repos/forcedotcom/dataloader/releases/latest` returns:
```
{
  "url": "https://api.github.com/repos/forcedotcom/dataloader/releases/56329766",
  "assets_url": "https://api.github.com/repos/forcedotcom/dataloader/releases/56329766/assets",
  "upload_url": "https://uploads.github.com/repos/forcedotcom/dataloader/releases/56329766/assets{?name,label}",
  "html_url": "https://github.com/forcedotcom/dataloader/releases/tag/v53.0.3",
  "id": 56329766,
  "author": {
    "login": "ashitsalesforce",
    "id": 71143047,
    ...
  "body": "This version is for use with Salesforce Winter '22 or higher release through Salesforce Force Partner API and Force WSC v53.0.0.  It  contains the fix for recent log4j vulnerabilities including CVE-2021-44832 by upgrading to log4j 2.17.1.
...
MD5 for dataloader_mac.zip: 72ea0d8baa7468ac6cd6bae4d059d641\r\nMD5 for dataloader_win.zip: 85a727f3cbc3051e1c8efddafcff0311\r\n[dataloader_mac.zip](https://github.com/forcedotcom/dataloader/files/7803572/dataloader_mac.zip)\r\n[dataloader_win.zip](https://github.com/forcedotcom/dataloader/files/7803574/dataloader_win.zip)\r\n"
}
```
Now users/sysadmins such as myself need to parse the _body_ instead of the _assets_ field to get the correct download URL.

Previous releases including [V53.0.2](https://github.com/forcedotcom/dataloader/releases/tag/v53.0.2) listed the downloads as _assets_; for example, `curl --silent --fail -H "Accept: application/vnd.github.v3+json" https://api.github.com/repos/forcedotcom/dataloader/releases` returns:
``` 
"assets": [
      {
        "url": "https://api.github.com/repos/forcedotcom/dataloader/releases/assets/52121674",
        "id": 52121674,
        ...
        "content_type": "application/zip",
        "state": "uploaded",
        "size": 20091120,
        "download_count": 1527,
        "created_at": "2021-12-19T06:58:01Z",
        "updated_at": "2021-12-19T07:01:12Z",
        "browser_download_url": "https://github.com/forcedotcom/dataloader/releases/download/v53.0.2/dataloader_mac.zip"
      }
```
Then the URL can be extracted via `curl --silent --fail -H "Accept: application/vnd.github.v3+json" https://api.github.com/repos/forcedotcom/dataloader/releases | awk -F '"' "/browser_download_url/ { print \$4; exit }"`.

Granted, the maintainers of this particular app case statement could use more error handling as it could break if the latest release is Windows-only and the first result, but that's a secondary issue.

**Comment by ashitsalesforce** [comment]
Hi @pythoninthegrass , thanks for bringing our attention to this aspect. Fixed the issue in v53.0.3 (added the 2 zip files to "Assets" and will observe this convention going forward.

**Comment by pythoninthegrass** [comment]
Thanks so much @ashitsalesforce! Works perfectly now ☺️
